autouri/gcsuri.py

```diff
@@ -166,6 +166,22 @@ def read(self, byte=False):
             return b
         return b.decode()
 
+    def find_all_files(self):
+        cl = GCSURI.get_gcs_client(self._thread_id)
+        bucket, path = self.get_bucket_path()
+        sep = GCSURI.get_path_sep()
+        if path != '':
```

Comment:
Why not just `if path`?
```suggestion
        if path:
```

Reply:
Fixed as you suggested.